packages/git-fetcher/src/index.ts

```diff
@@ -20,9 +21,13 @@ export default () => {
       const tempLocation = await cafs.tempDir()
       await execGit(['clone', resolution.repo, tempLocation])
       await execGit(['checkout', resolution.commit], { cwd: tempLocation })
-      await preparePackage(tempLocation)
       // removing /.git to make directory integrity calculation faster
       await rimraf(path.join(tempLocation, '.git'))
+      // invoke 'prepare' if package.json exists and defines a 'prepare' script
+      if (await exists(path.join(tempLocation, 'package.json'))) {
+        await preparePackage(tempLocation)
+      }
```

Comment:
Why was this moved down from line 23?

Reply:
I don't know :) I will move it back

Reply:
The lines are now in the original order. If you want, I could squash all commmits together? Otherwise it should be ready to merge.